.github/workflows/s3-production.yml

```diff
@@ -34,3 +34,6 @@ jobs:
           branch: release-hotfix
           base: main
           delete-branch: false
+      - name: Run 4everland deploy
```

Comment:
@mc2eqe please add this section before `name: Create Pull Request`. Pull request creation section was failing in the past. I am not sure if failed actions will stop the whole script, so move your code above to be on the safe side

Reply:
@bobo-k2 Change made with another commit. Please check.